napari/layers/points/points.py

```diff
@@ -362,31 +370,35 @@ def edge_width(self, edge_width: Union[None, float]) -> None:
 
     @property
     def edge_color(self) -> str:
-        """str: edge color of marker for the next added point."""
-
-        return self._edge_color
+        """Edge color of marker for the next added point."""
+        return hex_to_name.get(
+            self._edge_color.hex[0], self._edge_color.hex[0]
+        )
 
     @edge_color.setter
-    def edge_color(self, edge_color: str) -> None:
-        self._edge_color = edge_color
+    def edge_color(self, edge_color: ColorType) -> None:
+        self._edge_color = transform_color(edge_color)
         if self._update_properties and len(self.selected_data) > 0:
-            for i in self.selected_data:
-                self.edge_colors[i] = edge_color
+            cur_colors: np.ndarray = self.edge_colors.rgba
```

Comment:
again I don't love the syntax here, we're often going to and from the colorarray to the rgba array. If we just staid in rgba array land things might be much cleaner